src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -60,6 +60,7 @@
 (react/defc WorkspaceData
   {:render
    (fn [{:keys [props state refs]}]
+     ;(js* "debugger;")
```

Comment:
To share this knowledge without leaving cruft in the code, might be cool to make a utility function.